lib/reek/ast/sexp_extensions/constant_assignment.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+module Reek
+  module AST
+    module SexpExtensions
+      # Utility methods for :casgn nodes.
+      module CasgnNode
+        def send_expression
+          children[2]
+        end
+
+        def ancestor
```

Comment:
This method does not make sense for all constant assignments, so I think you should move its logic to the smell detector.